Vagrantfile

```diff
@@ -96,4 +96,16 @@ EOF
     cent.vm.provision "shell", inline: "systemctl start docker"
     cent.vm.provision "shell", inline: "systemctl enable docker"
   end
+
+  config.vm.define "ubuntu" do |ubuntu|
+    ubuntu.vm.box = "ubuntu/xenial64"
+
+    ubuntu.vm.provision "shell", inline: "apt-get update"
+
+    # Install required gems via Bundler
+    ubuntu.vm.provision "shell", inline: "apt install -y ruby-dev build-essential patch zlib1g-dev liblzma-dev"
+    ubuntu.vm.provision "shell", inline: "gem install bundler"
+    # ubuntu.vm.provision "shell", inline: 'sudo su -c "cd /home/vagrant/puppet-opendaylight; bundle install" vagrant'
```

Comment:
> This gives an error "ubuntu: No passwd entry for user 'vagrant'. Has to be solved.

This would happen if there wasn't a vagrant user. The command is trying to change to the vagrant user, running that command as them. I think the Ubuntu box ships with an ubuntu user, but I suspected vagrant would create a vagrant user as well.